### PR TITLE
Fix a few typos and formatting bugs in the docs

### PR DESCRIPTION
**Pull request by alexwlchan**

### CODE REVIEW

File: docs/getting-started/DigitalOcean.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ sudo iptables -A PREROUTING -t nat -i eth0 -p tcp --dport 80 -j DNAT  --to 192.1
 ```
 
 You should now be able to access the containers using the public IP address of your DigitalOcean host on port 80 by 
-visiting http://<host public ip>:80 or running:
+visiting http://`<host public ip>`:80 or running:
```
Comment (robbrockbank) [inline docs/getting-started/DigitalOcean.md]: I find just quoting the <host..> looks a bit odd when viewed, and we don't hightlight the <host...> in the curl below.

I'd either leave unchanged, or quote the entire http://blah

Comment (alexwlchan) [inline docs/getting-started/DigitalOcean.md]: I was adopting the style used in one of the other Getting Started docs. As it currently stands, those angle brackets are swallowed as an HTML tag. Feel free to change, but there's at least one other doc (not in this commit) where that style is used.

Comment (robbrockbank) [inline docs/getting-started/DigitalOcean.md]: Yeah I'll change to quote the entire http string - from a glance at other docs we don't just quote the angle-bracket parameter.  If we do that elsewhere I'll fix up when I see it.

**Comment by robbrockbank** [comment]
Thanks Alex,  I made a couple of minor tweaks and have now merged.

**Pull request closed without merging**